dotnet/src/Agents/Abstractions/AgentChat.cs

```diff
@@ -157,6 +178,8 @@ public void AddChatMessages(IReadOnlyList<ChatMessageContent> messages)
             // Note: Able to queue messages without synchronizing channels.
             var channelRefs = this._agentChannels.Select(kvp => new ChannelReference(kvp.Value, kvp.Key));
             this._broadcastQueue.Enqueue(channelRefs, messages);
+
+            this.Logger.LogInformation("[{MethodName}] Added Messages: {MessageCount}", nameof(AddChatMessages), messages.Count);
```

Comment:
Similar log in line 170 has level `Debug` while this log has level `Information` but messages are almost the same. Should they have the same log level in this case?

Reply:
Line 170 is "Add**ing**"

All the PRE messages (-ING) are DEBUG.  All the POST messages (-ED) are INFORMATION.

Reply:
![image](https://github.com/microsoft/semantic-kernel/assets/66376200/8386a1f8-8f91-4742-ae90-0a8a8e3673af)

Reply:
> All the PRE messages (-ING) are DEBUG. All the POST messages (-ED) are INFORMATION.

And what is the reason of that? In ASP.NET, default log level is `Information`. So, by default, using agents I will see only `post` messages with results, but not when operation started. In order to see that, I will have to explicitly enable `Debug` level. Why can't we keep both messages at the same level (e.g. `Debug` or `Information`) and control them at the same time (i.e. turn them on or off)? 

Just want to understand the benefits of this approach. And whether it's a common practice across other libraries. As a developer, I would expect to see `Started/Finished` messages at the same log level. 

From `LogLevel` documentation:
https://learn.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.loglevel?view=net-8.0#fields
![image](https://github.com/microsoft/semantic-kernel/assets/13853051/75fb4e49-e2a6-4c86-91c1-1736f13cb72b)

If we compare `Debug` and `Information`, the difference would be whether the message is useful during debugging and whether it has or hasn't long-term value. I'm not sure if we can differentiate `pre/post` messages based on these criteria, unless I'm missing something.